Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "mcp3425"
-version = "1.1.0"
+version = "2.0.0"
```

Comment:
This is a breaking change. E.g., my app uses `linux-embedded-hal` 0.4, which depends on `embedded-hal` 1.0. I cannot build my app with `mcp3425` 1.1.0, because versions 0.2 and 1.0 of `embedded-hal` are [incompatible](https://github.com/rust-embedded/embedded-hal/blob/520945278942c301433c391f63a075227d9e7c84/docs/migrating-from-0.2-to-1.0.md). I expect the opposite to be true as well: applications that use `linux-embedded-hal` 0.3 will not be able to compile with `mcp3425` after (or *if*) this patch is applied.